src/notation/view/internal/dynamicpopupmodel.h

```diff
@@ -0,0 +1,40 @@
+#ifndef MU_NOTATION_DYNAMICPOPUPMODEL_H
+#define MU_NOTATION_DYNAMICPOPUPMODEL_H
+
+#include <engraving/iengravingfontsprovider.h>
+#include <engraving/style/style.h>
+#include <view/abstractelementpopupmodel.h>
+
+#include <QList>
+#include <QObject>
+#include <QVariantList>
+
+using namespace mu::engraving;
```

Comment:
We prefer not using `using namespace` in header files, because that will affect all cpp files that directly or indirectly include this header, which may have unintended effects.

Instead, in header files, just write `engraving::` in full where necessary. However, it looks like it is only necessary for `IEngravingFontsProvider`, and it is already there, so you can just remove this `using namespace`.